dags/main_summary.py

```diff
@@ -387,6 +387,16 @@
     email=["telemetry-alerts@mozilla.com", "frank@mozilla.com"],
     dag=dag)
 
+search_rfm = bigquery_etl_query(
+    task_id="search_rfm",
+    destination_table="search_rfm_v1",
+    dataset_id="search",
```

Comment:
per https://github.com/mozilla/bigquery-etl/pull/782#pullrequestreview-369890788
```suggestion
    dataset_id="search_derived",
```